test/terraform_redeploy_example_test.go

```diff
@@ -152,7 +152,7 @@ func fetchSyslogForAsg(t *testing.T, awsRegion string, workingDir string) {
 
 	logger.Logf(t, "===== Syslog for instances in ASG %s =====\n\n", asgName)
 
-	for instanceId, logs := range asgLogs {
-		logger.Logf(t, "Most recent syslog for Instance %s:\n\n%s\n", instanceId, logs)
+	for id, logs := range asgLogs {
```

Comment:
Seems like `instanceID` would be a clearer name here than `id`

Reply:
You're right. I changed it back.